src/main/java/uk/gov/companieshouse/exemptions/controller/ExemptionsController.java

```diff
@@ -12,13 +11,9 @@
 import org.springframework.web.bind.annotation.RestController;
 import uk.gov.companieshouse.api.exemptions.CompanyExemptions;
 import uk.gov.companieshouse.api.exemptions.InternalExemptionsApi;
-import uk.gov.companieshouse.exemptions.model.CompanyExemptionsDocument;
 import uk.gov.companieshouse.exemptions.service.ExemptionsService;
-import uk.gov.companieshouse.exemptions.model.ServiceStatus;
 import uk.gov.companieshouse.logging.Logger;
 
-import java.util.Optional;
-
 @RestController
 public class ExemptionsController {
```

Comment:
Can remove the `@Autowired` annotations from lines 26 and 29 and instantiate the dependencies in a constructor. Also, can make the fields final too.